Tickets/preops 3838

leannep opened the pull request.

rblum5 commented on commissioning.tex:
```diff
@@ -1,27 +1,50 @@
-As a consequence, no on-sky data will be taken with the Commissioning Camera (ComCam). 
-This change in strategy will produce more early-commissioning data than would have been available with ComCam on a similar timeframe. 
+Following the October 2023 project schedule workshop, the Rubin Construction Project is reoptimizing the sequence of integration activities based on important updates to projected subcomponent availability.
+Given recent delays in the Camera ship data and important repairs needed to the summit dome crane, it will be beneficial to reinsert on-sky data taking with ComCam into the commissioning schedule for approximately two months, around July-August 2024. 
```
data->date

leannep replied:
already fixed

rblum5 commented on commissioning.tex:
```diff
@@ -1,27 +1,50 @@
+Given recent delays in the Camera ship data and important repairs needed to the summit dome crane, it will be beneficial to reinsert on-sky data taking with ComCam into the commissioning schedule for approximately two months, around July-August 2024. 
+This will enable Telescope commissioning, primarily the Active Optics System, to occur approximately four months earlier than could be done with LSSTCam. 
+With this updated strategy, System First Light with LSSTCam is projected to occur in January 2025, with Construction completion in May 2025.
+Without these changes, we predict an additional two months of delay.
```
I would delete this line. While true, It doesn't really make sense since of course we'd take longer if we don't adapt.

rblum5 commented on commissioning.tex:
```diff
@@ -1,27 +1,50 @@
-The System First Light milestone marks the end of the the on-sky engineering phase and the start of the System Optimization and Science Validation phases. 
+Figure~\ref{fig:commissioning-es-schedule} shows the detailed schedule of commissioning and early science activities relative to System First Light, as of October 2023.
+%Expected January 2025 - need to add the formula to code and  update the estimate here
+System First Light is currently expected in January 2025 (between XXX 2024 and YYY 2025) (\S~\ref{sec:timeline}), about 3 months after First Photon. 
```
There is still some confusion about first photon. Maybe just avoid using that term. 

"System First Light is currently expected in January 2025 (\S~\ref{sec:timeline}), about 3 months after on sky commissioning activities begin."

gpdf replied:
The term First Photon is defined explicitly above at line 9 (with reference to LSSTCam).  Are you suggesting removing the term altogether from this document?  I thought it actually did a good job of de-confusing the situation, by providing a change-controlled reference for the term.

leannep replied:
No  but due to the fact that we do not know whether First Photon will be with ComCam or LSSTCam, rather than define an additional term to disambiguate FP with CC/LSSTCam, I agree with Bob that his suggestion is less confusing

gpdf replied:
Got it, thank you.

bechtol replied:
My understanding is that if we have ~2 months of on-sky observations with ComCam, the expected time between First Photon and System First Light with LSSTCam is reduced from 12 weeks to 7 weeks because we expect to have made some progress with AOS commissioning using ComCam.

leannep replied:
Changed SFL to ~3 months to ~2 months  post FP in the text

rblum5 commented on commissioning.tex:
```diff
@@ -1,27 +1,50 @@
+%Expected January 2025 - need to add the formula to code and  update the estimate here
+System First Light is currently expected in January 2025 (between XXX 2024 and YYY 2025) (\S~\ref{sec:timeline}), about 3 months after First Photon. 
 The total amount of science validation time currently planned is about 8 weeks.  
 LSST data taking is expected to start about 4 months after System First Light.
```
With schedule uncertainty. LSST is 4-10 months after first light. 

"LSST data taking is expected to start 4-10 months after System First Light depending on construction schedue uncertainty and Operations readiness to start the survey following the start of full Survey Operations.

rblum5 replied:
With schedule uncertainty. LSST is 4-10 months after first light.

"LSST data taking is expected to start 4-10 months after System First Light depending on construction schedue uncertainty and Operations readiness to start the survey following the start of full Survey Operations.

leannep commented on commissioning.tex:
```diff
@@ -1,27 +1,50 @@
-As a consequence, no on-sky data will be taken with the Commissioning Camera (ComCam). 
-This change in strategy will produce more early-commissioning data than would have been available with ComCam on a similar timeframe. 
+Following the October 2023 project schedule workshop, the Rubin Construction Project is reoptimizing the sequence of integration activities based on important updates to projected subcomponent availability.
+Given recent delays in the Camera ship data and important repairs needed to the summit dome crane, it will be beneficial to reinsert on-sky data taking with ComCam into the commissioning schedule for approximately two months, around July-August 2024. 
```
```suggestion
Given recent delays in the Camera ship date and important repairs needed to the summit dome crane, it will be beneficial to reinsert on-sky data taking with ComCam into the commissioning schedule for approximately two months, around July-August 2024. 
```

leannep commented on commissioning.tex:
```diff
@@ -1,27 +1,50 @@
+Given recent delays in the Camera ship date and important repairs needed to the summit dome crane, it will be beneficial to reinsert on-sky data taking with ComCam into the commissioning schedule for approximately two months, around July-August 2024. 
+This will enable Telescope commissioning, primarily the Active Optics System, to occur approximately four months earlier than could be done with LSSTCam. 
+With this updated strategy, System First Light with LSSTCam is projected to occur in January 2025, with Construction completion in May 2025.
+Without these changes, we predict an additional two months of delay.
```
```suggestion

```

gpdf commented on datapreview.tex:
```diff
@@ -1,19 +1,22 @@
 \end{itemize}
 
-Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), these Data Previews will necessarily be limited in their area and temporal coverage relative to full a Data Release, however all Data Preview data products will be in the same science data model format as for future Data Releases.
+Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), these Data Previews will necessarily be limited in their area and temporal coverage relative to full a Data Release, however all Data Preview data products will be in the same science data model format as for future LSST Data Releases.
```
"... to a full Data Release ..."

leannep replied:
```suggestion
Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), these Data Previews will necessarily be limited in their area and temporal coverage relative to  a full Data Release, however all Data Preview data products will be in the same science data model format as for future LSST Data Releases.
```

gpdf commented on datapreview.tex:
```diff
@@ -1,19 +1,22 @@
 \label{sec:datapreview}
 
-A series of three Data Previews (DP) are planned  based on commissioning data to support the community as they develop their LSST analysis software and worfklows, and to enable high-impact science as soon as possible.
+A series of three Data Previews (DP) are planned  based on commissioning data to support the community as they develop their LSST analysis software and workflows, and to enable high-impact science as soon as possible.
```
I would remove "based on commissioning data" from this sentence, because it's not true for DP0, and the specifics are already in the bullet list that immediately follows.

leannep replied:
```suggestion
A series of three Data Previews (DP) are planned  to support the community as they develop their LSST analysis software and workflows, and to enable high-impact science as soon as possible.
```

gpdf commented on datapreview.tex:
```diff
@@ -1,19 +1,22 @@
 
 The data products that comprise a Data Preview are produced by the LSST Science Pipelines \citep{2019ASPC..523..521B,2018PASJ...70S...5B}.
 For an introduction to the LSST data products, see \citet{RubinDataProductsAbridged} and for a detailed description, see the LSST \dpdd{},  \citedsp{LSE-163}.
 Each pre-operations Data Preview and survey Data Release will be accompanied by its own release-specific DPDD\footnote{For an example data release DPDD, see the online DP0.2 documentation {\url{https://dp0-2.lsst.io/data-products-dp0-2/}}.}, giving e.g. the  database schema for the catalogs included in that dataset.
```
I would remove "pre-operations", as it could be read to exclude DP2, which is surely not what was intended.

leannep replied:
```suggestion
Each Data Preview and LSST Data Release will be accompanied by its own release-specific DPDD\footnote{For an example data release DPDD, see the online DP0.2 documentation {\url{https://dp0-2.lsst.io/data-products-dp0-2/}}.}, giving e.g. the  database schema for the catalogs included in that dataset.
```

leannep commented on commissioning.tex:
```diff
@@ -1,27 +1,50 @@
-The System First Light milestone marks the end of the the on-sky engineering phase and the start of the System Optimization and Science Validation phases. 
+Figure~\ref{fig:commissioning-es-schedule} shows the detailed schedule of commissioning and early science activities relative to System First Light, as of October 2023.
+%Expected January 2025 - need to add the formula to code and  update the estimate here
+System First Light is currently expected in January 2025 (between XXX 2024 and YYY 2025) (\S~\ref{sec:timeline}), about 3 months after First Photon. 
```
```suggestion
```

leannep commented on commissioning.tex:
```diff
@@ -1,27 +1,50 @@
+%Expected January 2025 - need to add the formula to code and  update the estimate here
+System First Light is currently expected in January 2025 (between XXX 2024 and YYY 2025) (\S~\ref{sec:timeline}), about 3 months after First Photon. 
 The total amount of science validation time currently planned is about 8 weeks.  
 LSST data taking is expected to start about 4 months after System First Light.
```
```suggestion
LSST data taking is expected to start 4-10 months after System First Light depending on construction schedule uncertainty and Operations readiness to start the survey following the start of full Survey Operations.
```

leannep commented on commissioning.tex:
```diff
@@ -1,27 +1,50 @@
+\subsection{Commissioning Milestones}
+\label{ssec:commissioning-milestones}
+
+Commissioning work is being planned around two major milestones, ''First Photon`` and ''System First Light``. 
```
```suggestion
Commissioning work is being planned around two major milestones, ''First Photon`` and ''System First Light``. 
```

gpdf commented on access.tex:
```diff
@@ -17,7 +17,7 @@ \subsection{Rubin Science Platform}
 The Portal and Notebook Aspects of the RSP make use of the same APIs as the API Aspect to internally access the LSST datasets.
 
-An early version of the RSP is currently running at the US Data Access Center (US DAC) that provides access to provides DP0.1 and DP0.2. 
+An early version of the RSP is currently running at the US Data Access Center (US DAC) that provides access to Data Preview 0. 
```
I've corrected a typo here and removed the specificity of reference to 0.1 and 0.2.  I think it's OK to say "running at the US Data Access Center" even though it blurs some details of differences between the DP0 data service architecture and the ultimate hybrid architecture.

gpdf commented on datapreview.tex:
```diff
@@ -105,7 +109,7 @@ \subsection{Data Release 1}
 \includegraphics[width=0.9\linewidth]{figures/DP1-products}
 \caption{Summary of data products expected in DP1.
-Note the high degree of uncertainty in this table. DP1 will be planned in detail during 2024.}
+Note the high degree of uncertainty in this table. DP1 will be planned in detail during 2024. \TODO{red}{NEEDS REVISION}}
```
I see the "NEEDS REVISION" but since the graphics here explicitly say "LSSTCam" perhaps it would help if the table caption were already updated to mention ComCam.

E.g.:

"DP1, which may be modified to be based on ComCam data, will be planned in detail during 2024."

leannep replied:
This is being updated still - the graphics require update

gpdf replied:
Ah, OK.  I had the impression you were trying to get a new version out before the graphics could be updated.  My suggestion would only be relevant in that case.

leannep commented on commissioning.tex:
```diff
@@ -1,27 +1,50 @@
-The System First Light milestone marks the end of the the on-sky engineering phase and the start of the System Optimization and Science Validation phases. 
+Figure~\ref{fig:commissioning-es-schedule} shows the detailed schedule of commissioning and early science activities relative to System First Light, as of October 2023.
+%Expected January 2025 - need to add the formula to code and  update the estimate here
+System First Light is currently expected in January 2025 (between XXX 2024 and YYY 2025) (\S~\ref{sec:timeline}), about 3 months after First Photon. 
```
```suggestion
System First Light is currently expected in January 2025 (\S~\ref{sec:timeline}), about 3 months after on sky commissioning activities begin.
```

rblum5 commented on datapreview.tex:
```diff
@@ -58,23 +61,24 @@ \subsubsection{Data Preview 0.2}
-Consequently DP1 has been redefined to be based on a subset of science-grade images taken with LSSTCam during a period of a few days around the System First Light milestone, \S~\ref{ssec:commissioning-observations}.
-The processing pipelines and exact data products that will comprise DP1 are still under discussion. 
-At minimum, DP1 will deliver visit-level images and catalogs to enable initial studies of observational and instrumental effects. 
+Data Preview 1 (DP1) will provide the community with a small set of useful science-grade data as soon as possible following System First Light (\S~\ref{ssec:commissioning-milestones}) to enable them to understand the characteristics of and prepare to work with LSST data. 
```
"Data Preview 1 (DP1) will provide the community with a small set of useful science-grade data as soon as possible following end of ComCam on sky and/or System First Light (DP1.1)..."

leannep replied:
```suggestion
Data Preview 1 (DP1) will provide the community with a small set of useful science-grade data as soon as possible following end of ComCam on sky and/or System First Light (\S~\ref{ssec:commissioning-milestones}) to enable them to understand the characteristics of and prepare to work with LSST data. 
```

leannep replied:
```suggestion
Data Preview 1 (DP1) will provide the community with a small set of useful science-grade data as soon as possible following end of ComCam on sky and/or System First Light (\S~\ref{ssec:commissioning-milestones}) to enable them to understand the characteristics of and prepare to work with LSST data. 
```

rblum5 commented on datapreview.tex:
```diff
@@ -58,23 +61,24 @@ \subsubsection{Data Preview 0.2}
-The processing pipelines and exact data products that will comprise DP1 are still under discussion. 
-At minimum, DP1 will deliver visit-level images and catalogs to enable initial studies of observational and instrumental effects. 
+Data Preview 1 (DP1) will provide the community with a small set of useful science-grade data as soon as possible following System First Light (\S~\ref{ssec:commissioning-milestones}) to enable them to understand the characteristics of and prepare to work with LSST data. 
+Depending on the commissioning schedule (\S~\ref{ssec:commissioning-schedule}) and  performance of the integrated system at the time, Data Preview 1 will be based either on ComCam  or early LSSTCam on-sky data. 
```
"Depending on the commissioning schedule (\S~\ref{ssec:commissioning-schedule}) and performance of the integrated system at the time, Data Preview 1 will be based on ComCam and/or early LSSTCam on-sky data."

leannep replied:
```suggestion
Depending on the commissioning schedule (\S~\ref{ssec:commissioning-schedule}) and performance of the integrated system at the time, Data Preview 1 will be based on ComCam and/or early LSSTCam on-sky data.
```

rblum5 commented on datapreview.tex:
```diff
@@ -58,23 +61,24 @@ \subsubsection{Data Preview 0.2}
+At minimum, both DP1 scenarios  will deliver visit-level images and catalogs to enable initial studies of observational and instrumental effects. 
+Table~\ref{tab:dp-one-products} presents a current best idea of  the data products that might  comprise DP1. 
 
+Note that the time period during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken around System First Light.
```
"Note that the time period during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken by the end of ComCam work on sky and/or around System First Light."

leannep replied:
```suggestion
Note that the time period during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken by the end of ComCam work on sky and/or around System First Light.
```

leannep replied:
```suggestion
Note that the time period during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken by the end of ComCam work on sky and/or around System First Light.
```

leannep replied:
```suggestion
Note that the time period during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken by the end of ComCam work on sky and/or around System First Light.
```

leannep replied:
```suggestion
Note that the time period during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken by the end of ComCam work on sky and/or around System First Light."
```

rblum5 commented on timeline.tex:
```diff
@@ -25,10 +41,9 @@ \section{Roadmap and Timeline} \label{sec:timeline}
 
-The next key milestone in the Early Science Program is the release of DP0.3. 
-Scheduled for mid-2023, DP0.3 will be the last in the DP0 series, which is based on simulated LSST-like data. 
+The next key milestone in the Early Science Program is the release of DP1
```
need period

leannep commented on timeline.tex:
```diff
@@ -25,10 +48,9 @@ \section{Roadmap and Timeline} \label{sec:timeline}
 
-The next key milestone in the Early Science Program is the release of DP0.3. 
-Scheduled for mid-2023, DP0.3 will be the last in the DP0 series, which is based on simulated LSST-like data. 
+The next key milestone in the Early Science Program is the release of DP1
```
```suggestion
The next key milestone in the Early Science Program is the release of DP1.
```

bechtol commented on commissioning.tex:
```diff
@@ -1,27 +1,50 @@
+The time from First Photon to System First Light is anticipated to be three months.
+During these three months, the focus will be on fine tuning the system including optical alignment and improving the image quality, collecting calibration data, and carrying out \textit{First Look} science programs. 
+
+For a detailed description of all the commissioning milestones and the most current dates, see \citeds{dmtn-232}
```
```suggestion
For a detailed description of all the commissioning milestones and the most current dates, see \citeds{dmtn-232}.
```

bechtol commented on commissioning.tex:
```diff
@@ -1,27 +1,50 @@
+
+\textbf {First Photon}: Defined as the very first photons to pass through the optical system and fall on the LSSTCam focal plane to produce an image of the night sky.
+
+\textbf {System First Light}: Defined as the first science-grade images of the night sky obtained with LSSTCam. 
```
We have been working on a more detailed definition of System First Light https://sitcomtn-061.lsst.io/v/u-kbechtol-init/index.html

Our working definition has been that System First Light is achieved when we can "routinely acquire science-grade imaging across the full focal plane, and have a well understood technical path towards meeting the Construction Completeness criteria"

leannep replied:
Integrated into text

leannep commented on commissioning.tex:
```diff
@@ -1,27 +1,50 @@
+
+\textbf {First Photon}: Defined as the very first photons to pass through the optical system and fall on the LSSTCam focal plane to produce an image of the night sky.
+
+\textbf {System First Light}: Defined as the first science-grade images of the night sky obtained with LSSTCam. 
```
```suggestion
\textbf {System First Light}: Defined as the point at which we can routinely acquire science-grade imaging across the full focal plane (LSSTCam) and have a well understood technical path towards meeting the Construction Completeness criteria   \citeds{sitcomtn-061}.
```

rblum5 commented on commissioning.tex:
```diff
@@ -1,27 +1,48 @@
+\subsection{Commissioning Milestones}
+\label{ssec:commissioning-milestones}
+
+Commissioning work is being planned around two major milestones, ``First Photon'' and ``System First Light''. 
```
This is more complicated with ComCam. If First photon is for LSTCam, then the period between FP and FL is less than 2 months (see dmtn-232). The text below needs to be reconciled with ComCam. It may be the easiest thing to do is not talk about first photon but just the activities.

leannep replied:
Added in the ComCam FirstLight milestone

rblum5 commented on commissioning.tex:
```diff
@@ -1,27 +1,48 @@
+
+Commissioning work is being planned around two major milestones, ``First Photon'' and ``System First Light''. 
+
+\textbf {First Photon}: Defined as the very first photons to pass through the optical system and fall on the  focal plane to produce an image of the night sky.
```
Formally only defined for LSSTCam.

gpdf replied:
I think it would assist users' comprehension if we just started saying "LSSTCam First Photon", throughout, then.  In particular that will help users understand that the 1.5-month period you suggest below doesn't include the time during which we'll be tuning up the AOS on ComCam.

leannep replied:
I originally understood "First Photon" to refer to the first photons through the optical system, irrespective of which camera they fell on. But I'm not sure now. If "First Photon" is to refer only to the first photons through the optical system and imaged by LSSTCam then we need another milestone ComCam on sky, as  a lot of the work that would be done after the original "LSSTCam First Photon" now gets done with ComCam, resulting in the shorter time between FP and SFL (as Gregory points out)

leannep replied:
All definitions now updated

rblum5 commented on commissioning.tex:
```diff
@@ -1,27 +1,48 @@
+There are no quality criteria applied to the images produced for the First Photon milestone. 
+First Photon  marks the start of the the on-sky engineering phase.
+System First Light  marks the end of the the on-sky engineering phase and the start of the System Optimization and Science Validation phases of commissioning.
+The time from First Photon to System First Light is anticipated to be three months.
```
If you change 3 months to 1.5 months, then maybe this is all ok.

leannep replied:
Updated paragraph

rblum5 commented on commissioning.tex:
```diff
@@ -1,27 +1,48 @@
-Figure~\ref{fig:commissioning-gantt} shows the detailed schedule of commissioning activities relative to System First Light, as of December 2022.
-System First Light is currently expected between October 2024 and February 2025 (\S~\ref{sec:timeline}), about 3 months after First Photon. 
-The System First Light milestone marks the end of the the on-sky engineering phase and the start of the System Optimization and Science Validation phases. 
+Figure~\ref{fig:commissioning-es-schedule} shows the detailed schedule of commissioning and early science activities relative to System First Light, as of October 2023.
```
The first block is inconsistent with ComCam. It is 6+months from ComCam on sky to FL. Or if this means FP with LSSTCam then that block is only 6 weeks. See dmtn232.

rblum5 commented on commissioning.tex:
```diff
@@ -1,27 +1,48 @@
-System First Light is currently expected between October 2024 and February 2025 (\S~\ref{sec:timeline}), about 3 months after First Photon. 
-The System First Light milestone marks the end of the the on-sky engineering phase and the start of the System Optimization and Science Validation phases. 
+Figure~\ref{fig:commissioning-es-schedule} shows the detailed schedule of commissioning and early science activities relative to System First Light, as of October 2023.
+System First Light is currently expected in January 2025 (\S~\ref{sec:timeline}), about 2 months after on sky commissioning activities begin.
```
```suggestion
System First Light is currently expected in January 2025 (\S~\ref{sec:timeline}), about 1.5 months after on sky commissioning activities begin with LSSTCam.
```

rblum5 commented on commissioning.tex:
```diff
@@ -1,27 +1,48 @@
+
+Commissioning work is being planned around two major milestones, ``First Photon'' and ``System First Light''. 
+
+\textbf {First Photon}: Defined as the very first photons to pass through the optical system and fall on the  focal plane to produce an image of the night sky.
```
```suggestion
\textbf {First Photon}: Defined as the very first photons to pass through the optical system and fall on the  LSSTCam focal plane to produce an image of the night sky.
```

leannep replied:
All definitions updated

rblum5 commented on commissioning.tex:
```diff
@@ -1,27 +1,48 @@
+There are no quality criteria applied to the images produced for the First Photon milestone. 
+First Photon  marks the start of the the on-sky engineering phase.
+System First Light  marks the end of the the on-sky engineering phase and the start of the System Optimization and Science Validation phases of commissioning.
+The time from First Photon to System First Light is anticipated to be three months.
```
```suggestion
The time from First Photon to System First Light is anticipated to be 1.5 months.
```

leannep replied:
Removed - we now have > 1 "First Photon"

rblum5 commented on commissioning.tex:
```diff
@@ -34,7 +55,7 @@ \subsection{Commissioning Observations}
 \centering
 \includegraphics[width=0.95\linewidth]{figures/commissioning-plan}
-\caption{Outline plan for the collection of commissioning data, as of December 2022.}
+\caption{Outline plan for the collection of commissioning data, as of October 2022.}
```
This figure needs to be updated

gpdf replied:
```suggestion
\caption{Outline plan for the collection of commissioning data, as of October 2023.}
```

rblum5 commented on datapreview.tex:
```diff
@@ -1,19 +1,20 @@
 \end{itemize}
 
-Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), these Data Previews will necessarily be limited in their area and temporal coverage relative to full a Data Release, however all Data Preview data products will be in the same science data model format as for future Data Releases.
+As both DP1 and DP2 will be based on commissioning data, they will not be released until after at least the 30-day embargo period for commissioning data has passed.
```
Suggest removing this. In all cases DPs take longer than 30 days to process and provide to community

gpdf replied:
Maybe it could be reworded, but if it isn't mentioned at all in this document, it's not clear where a community reader of this document would learn about the embargo policy.  Are we expecting a reader to have read DMTN-199 independently?  I think RTN-011 is quite a bit higher up the stack of what people are likely to read.

gpdf replied:
Or are you saying that, if we explicitly state latencies for DP1 and DP2 that are more than 30 days from the **last** data taken that's included in the DP, the embargo policy is simply irrelevant?  In that case perhaps we should say something like "No commissioning data will be released to data rights holders other than through these data previews"?

leannep replied:
The last statement is true in any case -- "No commissioning data will be released to data rights holders other than through these data previews".

leannep replied:
Reworked to simplify and include one reference to DMTN-199 covering all data products

rblum5 commented on datapreview.tex:
```diff
@@ -1,19 +1,20 @@
 
-Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), these Data Previews will necessarily be limited in their area and temporal coverage relative to full a Data Release, however all Data Preview data products will be in the same science data model format as for future Data Releases.
+As both DP1 and DP2 will be based on commissioning data, they will not be released until after at least the 30-day embargo period for commissioning data has passed.
+See \citeds{DMTN-199} for further details on the data embargo periods.
```
this can be removed too

rblum5 replied:
```suggestion
```

rblum5 commented on datapreview.tex:
```diff
@@ -1,19 +1,20 @@
 \end{itemize}
 
-Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), these Data Previews will necessarily be limited in their area and temporal coverage relative to full a Data Release, however all Data Preview data products will be in the same science data model format as for future Data Releases.
+As both DP1 and DP2 will be based on commissioning data, they will not be released until after at least the 30-day embargo period for commissioning data has passed.
```
```suggestion
```

rblum5 commented on datapreview.tex:
```diff
@@ -24,16 +25,16 @@ \section{Data Previews and Data Release 1}
 Data Preview 0 contains three stages, all based on simulated LSST-like data products. 
 The goals of DP0 are to serve as an early integration test of the LSST Science Pipelines and the Rubin Science Platform (RSP), and to enable a limited number of astronomers and students to begin early preparations for science with the LSST.
 
+The tables presented in the following  sections outline which data products can be expected in each Data Preview and Data Release, and on what time scale.
```
These need to be updated from DPRPT

leannep replied:
Tables all updated

rblum5 commented on figures/DP1-products.png:
these are updated in DPRPT

leannep replied:
I have removed these tables as the information os redundant with information in other tables

rblum5 commented on tables/milestones.tex:
```diff
@@ -0,0 +1,26 @@
+DP0.1  & Data Preview 0.1   &  DC2 Simulated Sky Survey                &  Delivered June 2021                  \\
+DP0.2 & Data Preview 0.2     &  Reprocessed DC2 Survey                 & Delivered June 2022                  \\
+DP0.3 &  Data Preview 0.3     &   Solar System PPDB Simulation       & Delivered July 2023                  \\
+DP1   &  Data Preview 1        &   ComCam data                                   & Oct 2024 -- Dec 2024  \\
```
```suggestion
DP1   &  Data Preview 1        &   ComCam data                                   & Oct 2024 -- Jul 2025  \\
```

gpdf commented on datapreview.tex:
```diff
@@ -1,19 +1,20 @@
-Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), these Data Previews will necessarily be limited in their area and temporal coverage relative to full a Data Release, however all Data Preview data products will be in the same science data model format as for future Data Releases.
+As both DP1 and DP2 will be based on commissioning data, they will not be released until after at least the 30-day embargo period for commissioning data has passed.
+See \citeds{DMTN-199} for further details on the data embargo periods.
+Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), these Data Previews will necessarily be limited in their area and temporal coverage relative to a full Data Release, however all Data Preview data products will be in the same science data model format as for future LSST Data Releases.
```
I think the statement "same science data model format as for future LSST Data Releases" is way too strong.  At most we should say "as for Data Release 1".  

There is no reason to use RTN-011 to make a long-range commitment like this.  I think it's highly likely that we'll learn things in the first year that will result in DR2 having a substantially improved data model compared to DR1 and the Data Previews.

I'm even going to suggest that we say something weaker still, like "Data Preview 1 and 2 data products will conform to the science data model planned for Data Release 1, though we reserve the right to then refine the DR1 data model based on project and user experience with the Data Previews."

leannep replied:
Agree, we need flexibility for the data model to evolve. My goal was more to communicate that the DP1/2 data products would be representative of what users would get in future DRs and hence useful to start to work with. I'll rephrase

leannep replied:
Removed completely  and simply stated that each release/preview will come with a specific DPDD

rblum5 reviewed (approved):
I approve this version. This version does not have the OPERATIONS time line from the PDFs tab of the DPRPT. I want to see that reinserted in the next version we release. Thanks.

bechtol commented:
Figure 1: I think it is closer to 7 weeks between LSSTCam First Photon and System First Light

bechtol commented on commissioning.tex:
```diff
@@ -4,37 +4,62 @@ \section{Rubin Observatory Commissioning}
-This change in strategy will produce more early-commissioning data than would have been available with ComCam on a similar timeframe. 
+Following the October 2023 project schedule workshop, the Rubin Construction Project is re-optimizing the sequence of integration activities in light of recent subcomponent delays, in particular, (i) delay in shipment of the Camera and (ii) necessary repairs to the  summit dome crane.
+If current estimates hold, it will be beneficial to re-implement on-sky data-taking with ComCam (previously removed from the schedule in order to install LSSTCam sooner).
+The updated plan calls for on-sky data to be taken with ComCam for approximately two months, around July-August 2024 to support Telescope commissioning ,primarily the Active Optics System. 
```
```suggestion
The updated plan calls for on-sky data to be taken with ComCam for approximately two months, around July-August 2024 to support Telescope commissioning, primarily the Active Optics System. 
```

leannep commented on tables/milestones.tex:
```diff
@@ -0,0 +1,26 @@
+DP1   &  Data Preview 1        &   ComCam data                                   & Oct 2024 -- Jul 2025  \\
+SFL    & System First Light    &   Science-Grade Images                      & Jan 2025 -- May 2025  \\
+OPS   & Start of Operations  & Operation Readiness Review  + 1 day & May 2025 -- Sep 2025 \\ 
+SVY   & Start of LSST           & Rubin Operation begins + 1 day  & May 2025 -- Sep 2025 \\
```
Thank Phil - this table has been replaced - I will make sure it is correct in the new table

leannep commented:
> @leannep @rblum5 RTN-011 looks almost good to go, nice work Leanne! I caught one bug: the DPRPT shows the late date for LSST survey start as being November 2025, up to two months after ORR, not September 2025 ("1 day after ORR"). I fixed this in both the relevent table and the text, via review comments with suggestions in them.

Thanks Phil - DP0.3 also needed updating -- it is delivered so the data range is no longer valid. I have replaced this table with a latex version now and fixed both these issues - thank you

leannep commented:
> Figure 1: I think it is closer to 7 weeks between LSSTCam First Photon and System First Light. That would match the text.

Fixed in new version

gpdf commented on datapreview.tex:
```diff
@@ -58,69 +52,42 @@ \subsubsection{Data Preview 0.2}
-Table~\ref{tab:dp-zero-three-products} presents a summary of the expected DP0.3 data products.
-The exact data products for DP0.3 are still to be decided. 
+Delivered in June  2023, DP0.3 is the last in the DP0 series of Data Previews based on simulated LSST-like data. 
+DP0.3 supports the Solar System Science Collaboration by hosting their simulated 1-year and 10-year Prompt Products Database )(PPDB) catalog to enable moving object analysis development  in  RSP at the US DAC. 
```
Stray ')' before '(PPDB)'

gpdf commented on datapreview.tex:
```diff
@@ -58,69 +52,42 @@ \subsubsection{Data Preview 0.2}
-The processing pipelines and exact data products that will comprise DP1 are still under discussion. 
-At minimum, DP1 will deliver visit-level images and catalogs to enable initial studies of observational and instrumental effects. 
+Data Preview 1 (DP1) will serve data products generated from a subset of science-grade astronomically-useful images as soon as 
+possible following the end of either ComCam on sky or System First Light (\S~\ref{ssec:commissioning-milestones}) to enable the community to prepare to work with LSST data. 
```
Run-on sentence.  Perhaps add commas at "images, as soon as" and "), to enable the"?

gpdf commented on datapreview.tex:
```diff
@@ -58,69 +52,42 @@ \subsubsection{Data Preview 0.2}
-At minimum, DP1 will deliver visit-level images and catalogs to enable initial studies of observational and instrumental effects. 
+Data Preview 1 (DP1) will serve data products generated from a subset of science-grade astronomically-useful images as soon as 
+possible following the end of either ComCam on sky or System First Light (\S~\ref{ssec:commissioning-milestones}) to enable the community to prepare to work with LSST data. 
+In the current schedule, which re-implements on-sky data-taking with ComCam for 2 months,  DP1 would be based on 2 months of ComCam data. 
```
"Re-implements" is jargon and sounds apologetic. Taking my next comment into account as well, maybe "In the current schedule, with ComCam used for initial on-sky commissioning, DP1 would be based on the best available data, from the end of approximately two months of ComCam data." ?

gpdf commented on datapreview.tex:
```diff
@@ -58,69 +52,42 @@ \subsubsection{Data Preview 0.2}
+Data Preview 1 (DP1) will serve data products generated from a subset of science-grade astronomically-useful images as soon as 
+possible following the end of either ComCam on sky or System First Light (\S~\ref{ssec:commissioning-milestones}) to enable the community to prepare to work with LSST data. 
+In the current schedule, which re-implements on-sky data-taking with ComCam for 2 months,  DP1 would be based on 2 months of ComCam data. 
+Note that the time period during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken by the end of ComCam work on sky or around System First Light.
```
I don't understand the "very short" -- is this really characterizing the "two months" from the previous sentence as "very short"?

leannep replied:
No - this is a hangover from when DP1 was to be a few nights of LSSTCam images. Good catch

leannep replied:
I will remove this as it's still uncertain as to what DP1 will be and I want to avoid setting any unrealistic expections

gpdf commented on datapreview.tex:
```diff
@@ -58,69 +52,42 @@ \subsubsection{Data Preview 0.2}
-Note that the DP1 period of time during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken in the few days around System First Light.
-Table~\ref{tab:dp-one-products} presents a summary of the data products expected in DP1.
+In both DP1 scenarios, the exact processing pipelines and data products are still under discussion.
+At minimum,  DP1  will deliver visit-level images and catalogs to enable initial studies of observational and instrumental effects. 
```
Do you want to say "calibrated" or "provisionally calibrated" images?

leannep replied:
Have we defined "provisionally calibrated" anywhere? All the data products in the DP program will be provisional, in that they are produced with in-development versions of the pipelines

gpdf commented on datapreview.tex:
```diff
@@ -58,69 +52,42 @@ \subsubsection{Data Preview 0.2}
+Data Preview 2 will serve a full consistent reprocessing of all data collected as part of the LSSTCam Science Validation Surveys (SV Surveys) together with any other science-grade 
+commissioning images taken throughout the Science Optimization phase of commissioning, including the DP1 data. 
+
+Table~\ref{tab:data-preview-summary} presents a summary of the data products expected in DP1.
```
I think "DP2" is meant here.

gpdf commented on datapreview.tex:
```diff
@@ -58,69 +52,42 @@ \subsubsection{Data Preview 0.2}
+Data Release Processing of this dataset is estimated to take six months, making the expected delivery date 1 year following the start of the 10 year survey. 
 DR1 will be the first Data Release in which all data products will be provided. 
 
 During routine LSST operations, prompt image data products will be made available 80 hours following camera readout.
```
I think having this sentence appear to be part of the DR1 subsection is misleading.  This paragraph is about Prompt Products, not DR1.

leannep replied:
removed

gpdf commented on timeline.tex:
```diff
@@ -1,39 +1,22 @@
 
-The next key milestone in the Early Science Program is the release of DP0.3. 
-Scheduled for mid-2023, DP0.3 will be the last in the DP0 series, which is based on simulated LSST-like data. 
+The next key milestone in the Early Science Program is the release of DP1.
```
It's odd that the next sentence gives a date window, for something further in the future, but this one does not.

gpdf reviewed (commented):
I've made some comments.  None of them are "on fire"-level, so I'm not using "request changes".

I will also read the tables in the PDF, which aren't so amenable to inline commenting.

gpdf commented on RTN-011.tex:
```diff
@@ -129,7 +129,7 @@
   \addtohist{13}{2023-06-01}{Updates following April schedule workshop; revision of early science scenarios; additional details on the data access environment}{Leanne Guy}
   \addtohist{14}{2023-06-21}{Release version 4.0}{Leanne Guy}
-
+  \addtohist{15}{2023-10-31}{Updates following April schedule workshop}{Leanne Guy}
```
"October"

gpdf commented on tables/rubin_early_dr_scenario.tex:
I think there's something seriously wrong and misleading about this table, unless I'm really misunderstanding it.

The rows with headings beginning with "PP" should not be checked off in _any_ Data Preview or Data Release.

Prompt Products are _by definition_ not included in Data Releases and all of our Data Previews _including DP0.3_ have been DRP-like.

gpdf replied:
If you are comfortable adding a column describing what _Prompt_ data products will start to be available at the start of formal operations (which I think would be a good idea, pedagogically), that's the place to mark off the "PP" items.

gpdf commented on datapreview.tex:
```diff
@@ -34,6 +25,9 @@ \subsection{Data Preview 0}
 The goals of DP0 are to serve as an early integration test of the LSST Science Pipelines and the Rubin Science Platform (RSP), and to enable a limited number of astronomers and students to begin early preparations for science with the LSST.
 
+The following sections outline which data products can be expected in each Data Preview and Data Release, and on what time scale.
+See Table~\ref{tab:milestones} in the Timeline section below for a combined view of the expected data preview schedule and its uncertainties.
```
This reference is broken in the just-released version.

leannep replied:
Now fixed

leannep commented:
Additional comments not addressed will be addressed in the next issue

gpdf commented:
> Additional comments not addressed will be addressed in the next issue

I am fine with that and I will carry them forward to a new ticket/PR.

leannep commented on commissioning.tex:
```diff
@@ -4,37 +4,62 @@ \section{Rubin Observatory Commissioning}
+System First Light is currently expected in January 2025 (\S~\ref{sec:timeline}), about 7 weeks after LSSTCam First Photon.
 The total amount of science validation time currently planned is about 8 weeks.  
-LSST data taking is expected to start about 4 months after System First Light.
+LSST data taking is expected to start 4-10 months after System First Light depending on construction schedule uncertainty and Operations readiness to start the survey following the start of full Survey Operations.
```
This is meant to reflect the fact that the start of the Operations era, i.e when responsibility is handed from the Construction project to Operations, is not necessarily the start of the survey

leannep replied:
I'll remove the last part - that hopefully will make it clearer

leannep commented on commissioning.tex:
```diff
@@ -4,37 +4,62 @@ \section{Rubin Observatory Commissioning}
+\subsection{Commissioning Milestones}
+\label{ssec:commissioning-milestones}
+
+Commissioning work is being planned around three major milestones, \textit{ComCam First Photon}, \textit{LSSTCam First Photon'} and \textit{System First Light}. 
```
fixed

leannep commented on commissioning.tex:
```diff
@@ -4,37 +4,62 @@ \section{Rubin Observatory Commissioning}
+
+\textbf {System First Light}: Defined as the point at which we can routinely acquire science-grade imaging across the LSSTCam full focal plane and have a well understood technical path towards meeting the Construction Completeness criteria   \citeds{sitcomtn-061}.
+Also referred to as \textbf{First Light}. 
+These images will be shared with the worldwide community. 
```
Removed it -- The topic requires more detail, that will be added later
